### DISCUSSION TITLE
How to Configure Promethus to Pull/Scrape Kafkaexporter Metrics

### DISCUSSION THREAD

**Discussion by jeffreywstevens** [strimzi/strimzi-kafka-operator#7553] [General]
I am new to all of this. However, I have the KafkaExporter configured, and it is spewing good metrics at the http://kafka-exporter:9404/metrics endpoint. 

How do I configure prometheus to pull these metrics and then make them available to Grafana?

**Comment by scholzj** [comment]
It depends on your Prometheus installation. There is no single way which applies everywhere. For example - if you use the Prometheus Operator - you can create the `PodMonitor` custom resource as used in our examples. But as I said, it really depends on how you installed and configured Prometheus in the first place.

**Reply by jeffreywstevens** [reply]
I am trying to find the most reliable, repeatable process to prepare for a future prod installation of Kafka with monitoring. I have found Strimzi the best way forward versus Confluent's.  I am trying to do the same for Prometheus. The Kube-prometheus operator appears to be a solid installation. https://github.com/prometheus-operator/kube-prometheus  . 

So I have the Strimzi metrics up.  Prometheus and Grafana are running the k8s cluster.  Grafana is connected to Prometheus's data source. I need to be able to feed in the Strimz metrics and show them on a dashboard. This will be a win!

**Reply by scholzj** [reply]
So, did you checked our examples and the docs? As they are also based on the Prometheus Operator.

**Comment by jeffreywstevens** [comment]
Since your metric endpoint works, I was hoping someone could point me to an easy way to add a target to prometheus to scrape your endpoint. I will read through your docs some more.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
